src/components/Crews/ResultOverview.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+    <b-container v-if="crews.length">
+        <div class="text-center mt-3">
+            <h3 class="text-secondary">{{ fieldName }}</h3>
+        </div>
+        <b-row class="mt-2">
+            <b-table
+                hover
+                :items="crews"
+                :fields="fields"
+                thead-class="thead-dark"
+                @row-clicked="openTeamDialog"
+            >
+                <template slot="OarImage" slot-scope="row">
+                    <OarImage :club="row.item.clubnameshort"></OarImage>
+                </template>
+                <template slot="FinishTime" slot-scope="row">
+                    <FinalTime
+                        :time="row.item.times[0].times[0].time"
```

Comment:
Volgens mij geeft dit met tussentijden nu de 1e tussentijd ipv de finish tijd.

Reply:
Zeker waar voor de iframe is dit niet direct een probleem

Reply:
Zou je hier niet times[times.size -1] kunnen doen voor altijd de laatste tijd?